Documentation/network/servicemesh/istio.rst

```diff
@@ -36,35 +36,45 @@ You can confirm the result with the following command:
     $ kubectl get configmaps -n kube-system cilium-config -oyaml | grep bpf-lb-sock-hostns
     bpf-lb-sock-hostns-only: "true"
 
+
+To ensure that Cilium does not interfere with other CNI plugins on the node, it is important to set the
```

Comment:
`with other CNI plugins on the node` Since this doc is specific to Istio integration, should this general statement be more specific by stating "with the Istio CNI plugin"

Reply:
The flag itself is general, and exists for any CNI (AWS, etc) - any other CNI plugin would have this problem without this flag, and other parts of the doc for other CNI/cloud integrations have the same guidance for the same reason.